README.md

```diff
@@ -11,10 +11,31 @@ for help getting started.
 Visit the [Servo Project page](https://servo.org/) for news and guides.
 
 ## Getting Servo
+The first step is to `fork the repository` to your GitHub account. After forking, proceed to clone the repository using the instructions provided below.
+
 ``` sh
-git clone https://github.com/servo/servo
+git clone https://github.com/<your username >/servo.git
+```
+
+````{note}
+To address potential barriers with slow internet connections due to the large size of this repository, you can use [partial clones](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/#). Specifically, focusing on blobless clones, which are efficient for developers, involves utilizing the `--filter=blob:none` option in the git clone command.
+
+By using `--filter=blob:none`, the initial git clone operation downloads all reachable commits and trees, while blobs (file contents) for commits are only downloaded when performing a git checkout
+
+Here's the command to use blobless clones:
+
+```console
+git clone --filter=blob:none https://github.com/<your username>/servo.git
+
+```
+````
+
+> Navigate into the cloned repository folder 
+
+```console
 cd servo
 ```
+
```

Comment:
1. I don't think we want to say that the first step to getting servo is forking the repository. That doesn't work for users that aren't using GitHub.
2. Instead of copying the instructions for doing a shallow clone here, which we'd have to maintain, why don't we just link to GitHub's instructions for those facing network issues.

Here's an idea for how to do this change:

````markdown
## Getting Servo

```sh
git clone https://github.com/servo/servo
```

 - Your CARGO_HOME needs to point to (or be in) the same drive as your 
   Servo repository ([#28530](https://github.com/servo/servo/issues/28530)).
 - The Servo repository is big! If you have a slow network, considering [making
   a shallow clone](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/).
 ````